### ISSUE TITLE
Enable translations

### ISSUE THREAD

**Issue by alexgleason** [HarvestHub/GardenHub#109]
While I realize this isn't really pertinent, it's trivial enough to do this that I see no reason not to. All we have to do is find every string in the code and wrap it like so: `_('my special words')`. In templates, there's a `{% trans %}` tag which does the same.

This would enable people to translate the application into multiple languages.

**Comment by marykatefain** [comment]
We'd have to do this to EVERY string? Will this be hard to maintain?

**Comment by alexgleason** [comment]
Yes, and not really. Full docs are here. https://docs.djangoproject.com/en/2.0/topics/i18n/translation/

I think it's good to just get into the habit of doing this in the first place. Writing a string? Wrap it in the translation class.
